tools/Environments/DevHome.Environments/Extensions/ServiceExtensions.cs

```diff
@@ -17,9 +17,6 @@ public static IServiceCollection AddEnvironments(this IServiceCollection service
         // View-models
         services.AddSingleton<LandingPageViewModel>();
 
-        // Services

```

Comment:
This was only used for a Debug only testing scenario. Removing as it is now out of date.